slam-libraries/viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -764,6 +778,14 @@ void ParseAndValidateArguments(const vector<string> &args,
         BOOST_LOG_TRIVIAL(info) << "No camera given -> running in offline mode";
         slamService.offlineFlag = true;
     }
+
+    auto delete_processed_data = ArgParser(args, "-delete_processed_data=");
+    slamService.delete_processed_data = !(delete_processed_data == "false");
```

Comment:
This will only change the value from the default of true if false is given. If anything else, say 'blahblahblah' is given, value will remain true. Could add an additional error throwing check if the value of `delete_processed_data != 'false' || delete_processed_data != 'true'`